pkgs/development/libraries/mlt/default.nix

```diff
@@ -31,6 +31,7 @@
 , swig
 , enableQt ? false
 , libsForQt5
```

Comment:
Redundant given the next line?

Reply:
Technically we could make it `qt = null`, yeah.